lib/presentation/models/word_cloud/word_cloud_data.dart

```diff
@@ -0,0 +1,78 @@
+import 'package:flutter/painting.dart';
+import 'dart:ui';
+
+class FlutterHashtag {
+  const FlutterHashtag(
+    this.hashtag,
+    this.color,
+    this.size,
+    this.rotated,
+  );
+  final String hashtag;
```

Comment:
should also have a more generic title like `name`

Reply:
Addressed in commit 7a8dd1bb251c728c822740f7faf2f8499c6bd46b